Formula/activemq.rb

```diff
@@ -14,6 +14,32 @@ def install
     (bin/"activemq").write_env_script libexec/"bin/activemq", Language::Java.java_home_env("1.6+")
   end
 
+  plist_options :manual => "activemq start"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>AbandonProcessGroup</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{libexec}</string>
```

Comment:
This should have probably been `opt_libexec`, or we're going to have the same problem where the plist needs unloading, replacing & reloading every revision/update.